pravega/pravega-operator
# Issue-316-added support for DomainName changes

Signed-off-by: prabhaker24 <prabhaker.saxena@dell.com>

### Change log description
This pr allows the changes in DomainName after the cluster is deployed and changes the **publishedIPAddress** for the pod post the change in the **domainName** filed in the cluster. 

### Purpose of the change
Fixes #316 #463 

### What the code does
This Pr allows the user to change the domainName once the cluster is installed and will change the publishedIPAddress of the pod after that.

### How to verify it
1) Install a pravega cluster without the domain name and update it. It should change the publishedIPAddress  for the ss pods post that
2) Install a pravegacluster with some domainName and update the domainName  It should change the published IP address  for the ss pods post that.
I have checked both the scenarios.
